### ISSUE TITLE
Prefilter Cubemap - Uncaught TypeError: Cannot read property ‘format’ of undefined

### ISSUE THREAD

**Issue by Mentos-** [playcanvas/editor#171]
When I try to prefilter a cubemap on any project the loading bar waits for about 30 seconds then fails with Uncaught TypeError: Cannot read property ‘format’ of undefined

![prefilterCubemapIssue](https://user-images.githubusercontent.com/3871989/94992995-ccbc7e00-055b-11eb-99dd-c8345db1e621.png)

Here’s a demo project https://playcanvas.com/editor/scene/994352

Not sure what I might have setup incorrectly or if maybe this is just some part of the playcanvas service that is down?

**Comment by slimbuck** [comment]
Hi @Mentos- thanks for this I can repro.

**Comment by slimbuck** [comment]
Unfortunately our tools handle HDR texture import in a strange way (for historical reasons).

When HDR files are imported to a project, we actually generate and keep an invisible Float32 version of the HDR file on the server for use when calculating prefiltered lighting data (we don't use the HDRs directly or the generated PNG files).

When a project is forked however, we don't copy those invisible files with and so prefiltering fails. (This is what's happening in your case).

You could get this to work by downloading the HDR textures, then deleting them from your project and re-imported them.

We will use this ticket to fix our backend and get prefiltered lighting to work post-fork.

Thanks!

**Comment by Mentos-** [comment]
Hey @slimbuck I was able to reproduce this issue in a new project that was not forked by enabling rgbm on the cubemap's source textures: https://playcanvas.com/editor/scene/1002802

I downloaded the room_back.png,room_down.png,room_front.png,room_left.png,room_right.png,room_up.png from the forked project, created a new project, uploaded those textures, created a new cubemap, set the slots to each texture then hit the 'Prefilter Cubemap' button, it successfully created the data. I then deleted the cubemap data, refreshed the page, enabled rgbm on each texture to match the original examples texture settings, refreshed the page again, then attempting to 'Prefilter Cubemap' gave me the 'Uncaught TypeError: Cannot read property ‘format’ of undefined' after 30 seconds.

Not sure if that helps shine any extra light on the problem.
